integration/integration_test.go

```diff
@@ -3262,9 +3262,7 @@ func testAuditOff(t *testing.T, suite *integrationTestSuite) {
 	}
 
 	// audit log should have the fact that the session occurred recorded in it
-	sessions, err = site.GetSessions(apidefaults.Namespace)
-	require.NoError(t, err)
-	require.Len(t, sessions, 1)
+	// but the session could have been garbage collected at this point.
```

Comment:
Hmm, is this change related to this PR? (Aside from it failing a PR check)

Reply:
Indirectly. The race between this test and the garbage collection existed beforehand, but the test happened to pass regularly. The `isAuthServerBoring()` check slows the test down enough so that it fails regularly instead.